dynatrace_extension/sdk/communication.py

```diff
@@ -322,16 +322,26 @@ class DebugClient(CommunicationClient):
     def __init__(
         self,
         activation_config_path: str,
+        secrets_path: str,
```

Comment:
Make this optional please so we don't make a breaking change here

Reply:
Fixed in latest PR